### PR TITLE
completed hw 3 with 3 buttons and onlclick on projects menu

### PR DESCRIPTION
**Pull request by aberezhk**
Added an html page with 3 buttons
Added onclick event for projects menu that displays an alert

### CODE REVIEW

**Review by caritosteph — APPROVED** [review]
Nice!
Looks good for me.

File: html/buttons-on-click.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        <button id="badButton">Bad 👎</button>
+        <p id="answer"></p>
+    </div>
+</div>
+
+<div class="footer-bar">
+    <span> Demo project for spring 2020 semester JS class at ReDI Munich </span>
+</div>
+
+<style>
```
Comment (MrTim) [inline html/buttons-on-click.html]: Why we have inline styles here? why not in css folder? The same for js.

File: html/buttons-on-click.html [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+</style>
+
+<script src="../js/main.js"></script>
+<script>
+    let awesomeButtonCmp = document.getElementById("awesomeButton");
+    let goodButtonCmp = document.getElementById("goodButton");
+    let badButtonCmp = document.getElementById("badButton");
+    let answerCmp = document.getElementById("answer");
+
+    function awesomeButtonClick(event) {
```
Comment (MrTim) [inline html/buttons-on-click.html]: event is not used so I would omit it. Sometimes I feel like eslint :-)

**Review by MrTim — APPROVED** [review]

File: html/buttons-on-click.html (hunk 1 above)
Comment (aberezhk) [reply html/buttons-on-click.html]: The idea was to keep it similar to how it was presented in class, but sure, i can move it to css and js files

Comment (aberezhk) [reply html/buttons-on-click.html]: done

File: html/buttons-on-click.html (hunk 2 above)
Comment (aberezhk) [reply html/buttons-on-click.html]: done

**Pull request merged**